api/handlers/metadata.go

```diff
@@ -60,45 +63,54 @@ type metadataHandler struct {
 func (h *metadataHandler) getMetadata() middleware.Responder {
 	logger.Info("API received a GET metadata event")
 
-	var namespace string
-	namespace = os.Getenv("POD_NAMESPACE")
+	namespace := os.Getenv("POD_NAMESPACE")
 
 	var payload models.Metadata
 	payload.Namespace = namespace
-	payload.Keptnversion = "N/A"
+	payload.Keptnversion = defaultVersion
 	payload.Keptnlabel = "keptn"
-	payload.Bridgeversion = "N/A"
+	payload.Bridgeversion = defaultVersion
 	payload.Shipyardversion = "0.2.0"
 
 	if h.k8sClient != nil {
 		deploymentsClient := h.k8sClient.AppsV1().Deployments(namespace)
-		bridgeDeployment, err := deploymentsClient.Get(context.TODO(), "bridge", metav1.GetOptions{})
-		if err != nil {
-			// log the error, but continue
+
+		if bridgeDeployment, err := deploymentsClient.Get(context.TODO(), "bridge", metav1.GetOptions{}); err == nil {
```

Comment:
since the retrieval of getting the swagger keptn version is extracted into a method, it should also be done for getting the bridge version, to keep the level of detail equal...
maybe something like this:
```go
func (h *metadataHandler) getBridgeVersion(namespace string) (string, error) {
	if h.k8sClient == nil {
		return "", fmt.Errorf("unable to get bridge version")
	}
	bridgeDeployment, err := h.k8sClient.AppsV1().Deployments(namespace).Get(context.TODO(), "bridge", metav1.GetOptions{})
	if err != nil {
		return "", fmt.Errorf("unable to get bridge version %w", err)
	}

	v := strings.Split(bridgeDeployment.Spec.Template.Spec.Containers[0].Image, ":")
	if len(v) >= 2 {
		return v[1], nil
	}

	return "", fmt.Errorf("unable to get bridge version")
}
```